### PR TITLE
Remove @znewman01 :'(

### PR DESCRIPTION
**Pull request by znewman01**

### CODE REVIEW

**Comment by cpanato** [comment]
Thank you for all your work and support! I've learned a lot from you.

**Comment by cpanato** [comment]
After the core team approve, I will remove the block from the automation to be able to merge this

**Review by bobcallaway — APPROVED** [review]
😞

**Pull request merged**
